.gitignore

```diff
@@ -2,3 +2,4 @@ static/compiled
 node_modules
 npm-debug.log
 bower_components
+*.idea
```

Comment:
I think it’s best to use a [global excludes file](https://help.github.com/articles/ignoring-files/) to keep out artifacts generated by the editor of your choice—hence no `.swp` etc. in here even though I’m a vim user. Would that work for you?